src/components/index.js

```diff
@@ -0,0 +1,7 @@
+import CreatureList from './CreatureList';
+import StateManagement from './StateManagement';
+
+export {
+  CreatureList,
+  StateManagement
+}
```

Comment:
index.js files act as middle-men for cross-domain dependency (eg. the `global` domain App which depends on the `component` domain CreatureList and StateManagement). This gives us some intentionality and some functionality.
The intention is that the creature and counter components are, presumably, private and local to the components domain. The functionality is that modules in other domains don't need to know about the potentially complex filesystem of other domains.